airbyte-workers/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation project(':airbyte-scheduler:models')
 
     testImplementation 'io.temporal:temporal-testing:1.8.1'
-    testImplementation 'io.temporal:temporal-testing-junit5:1.5.0' // versioned separately from rest of temporal
```

Comment:
what happened to this?

Reply:
I think that it is not needed. The `temporak-testing-junit5` version doesn't go above 1.5 we should only keep the `temporak-testing` to be align with the temporal sdk

Reply:
sounds good.